beamerthemeuulm.sty

```diff
@@ -123,10 +123,9 @@
 \universitylogo{uulm}
 \clearinstitutelogo
 
-\renewcommandx{\maketitle}[2][1=apr21-o25a,2=150]{
+\newcommand{\createtitleslide}[2]{
     {
-    \def \picture {#1}
-    \ifx \picture \empty \else \usebackgroundtemplate{\includegraphics[trim=0 0 0 #2,clip,width=\paperwidth]{#1}} \fi
+    \ifx \uulmtitlepicture \empty \else \usebackgroundtemplate{\includegraphics[trim=0 0 0 #2,clip,width=\paperwidth]{#1}} \fi
```

Comment:
Sadly, GitHub did not allow me to make suggestions for this segment because 1) it would have affected multiple disjoint positions and 2) always included delete lines which apparently can not be part of a suggestion.

Reply:
If i use `#1` instead of `\uulmtitlepicture` (either with `\ifx` or with `\iftstrempty`) i get the following error message: **File `' not found**

Reply:
However i could just use the global definitions of `\uulmtitlepicture` and `\uulmtitlepictureoffset` in the definition of `\createtitleslide` and not give it any parameters.

Reply:
That would be the first variant of my comment :D

Reply:
You're right, i didn't see that you removed the parameters :) I implemented this change.